build/main.js

```diff
@@ -0,0 +1,1889 @@
+// elm-watch hot {"version":"1.0.2","webSocketPort":59209}
```

Comment:
This file isn’t supposed to be committed, is it?

Reply:
well spotted once again, my bad, i removed it 👍